src/styles/colors.js

```diff
@@ -7,9 +7,8 @@ const colors = {
   blue06: '#006DF0',
   blue05: '#2488FF',
   blue04: '#84DBFF',
-  blue03: '#9DBDFD',
-  blue02: '#002740',
-  blue01: '#005D9D',
+  blue061: '#9DBDFD',
+  blue062:  '#002740',
```

Comment:
I think the names should be more intuitive

Reply:
All color are defined on the same way, so I think I shouldnt change that.
![image](https://user-images.githubusercontent.com/92061563/156682548-790b3102-b51a-4c84-9baf-1a6d010cbec8.png)


There is another file where colors are named more properly
![image](https://user-images.githubusercontent.com/92061563/156682749-7ac918f4-eafc-4c8a-8ee1-fdb0b15c7d80.png)

Reply:
I refer to blue061, I don't understand why the number selected is "061" and "062", What do you think about blue12 and blue13? to continue with the counter (10, 11.. 12, 13)

Reply:
I understand now. Its done.

Reply:
Thanks!